docs/reference/upgrade/rolling_upgrade.asciidoc

```diff
@@ -120,6 +120,8 @@ GET _cat/health
 
 Wait for the `status` column to switch from `yellow` to `green`. Once the
 node is `green`, all primary and replica shards have been allocated.
+(If the node you removed contained any shards for which there are no
+replicas in the cluster, the status will start as `red` rather than `yellow`.)
```

Comment:
While technically accurate, the phrasing seems to imply that removing a node is a normal part of a rolling upgrade. To my knowledge, that's not the case.

Can you explain a bit more @GlenRSmith?